### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by raccettura** [wanderinggnome648/Nightingale#2]
This looks pretty good.

Wasn't sure the best way to provide some feedback, so figured I'd just open an issue...  I've been looking into building something similar but perhaps 8 fans and using an esp32 running @esphome to make the fan speed something that could be controlled and monitored via @homeassistant.  A few thoughts popped up aside from sizing:

1. Why not move the fans inside the case and put the fan guards on the exterior?  That would certainly improve the "curb appeal" of the device.
2. Why not add some weather seals around the air filter for sure, possibly also the fans?  That would ensure more airflow is going through the filter rather than around it and should improve effectiveness.

**Comment by wanderinggnome648** [comment]
Cool. No worries, I'm actually still learning these interfaces, too. Few thoughts. 
1. My early designs put the fan inside, where it's also convenient for the fan to rest against. I changed it after some early designs that seemed like they should have been good weren't that good using the salt crystal particle counting method (shoot pure salt water in the air with a nebulizer and then track with a particle counter how quickly your purifier removes it in a confined space like a bathroom. I am not a fluid dynamics expert, but after many iterations and comparing against other purifier designs, I learned that the interior size needed to be minimized. Also, I learned that I couldn't just deck out the box with fans, but in certain positions (e.g., across from each other), they interfered. I also found a few papers and tweets on similar topics. On the fan placement inside vs. outside, I guessed that outside might be superior based on the Air Fanta 3Pro's design. My suspicion is that pulling the air out is better than pushing it out, but I haven't confirmed this in a head-to-head test. But that's why I decided to default to outside and use the 3d printed holder on the interior. To be clear, it's a guess. 
2. Again, earlier in my tests I did a lot of sealing with tape and foam when I believed that was a problem. Turned out, it was my designs that were bad and the sealing didn't matter. At the moment, I don't think I'm losing a ton of air through the small gaps. However, when I do my close-to-final test, I'll probably try that, too, to see if it makes a difference. Other makers in this space seem to have the same belief -- as long as the design is decent, the small gaps don't seem to matter. 

On the monitoring/speed control, I love programmable microcontrollers and I debated on whether or not something like that was useful. For my specific use case of something for classrooms, I've avoided adding anything extra due to cost. What I may consider down the road is LoRaWan for monitoring. My rationale is that for schools, I want to know that it's being used, but I can't trust (or expect) the school to give my purifier wifi access. So, instead, I can just broadcast the stats via LoRaWan and have a nearby parent save and relay the data. So, my only thoughts for microcontrollers are just thinking through your use case and whether you might want to scale it up and just make sure it makes sense. 

On the multiple fans, my advice is to mock up different designs with cardboard, fans, filters, and duct tape. And do quick tests with salt particles to see how well it works. It was humbling for me to see good design ideas I had didn't work well in practice. The Corsi box design (and AirFanta 3Pro) works well (to my current understanding) because it's a box with a lot of filter area, somewhat minimized interior space, and all the fans on one side pulling in the same direction. The Tempest 6 fan design (3 fans at right angles) seems to work well because it's a good amount of filter area, minimized interior space, and 6 fans pulling in non-competing directions. 

Hope that helps and good luck with your builds.

**Comment by raccettura** [comment]
Thanks for the info.  

Researching I've also seen cleanairkits's "luggable" designs which are pretty similar but 120mm internally mounted fans.  That's a bit cleaner of a design, but looks somewhat flimsy, I like the frame you're using, and definitely would want to go that route. My current thinking is to combine the two approaches with internally mounted fans. Their CADR seems pretty impressive based on the data in their [faq](https://www.cleanairkits.com/pages/frequently-asked-questions) leading me to believe it should work if the design is snug enough.

**Comment by wanderinggnome648** [comment]
Yes, the sturdiness of the frame is a downside. I have 2 towers of power and my dad has a luggable. If nothing else, a sturdier frame is a huge win. Big thing for me was thinking long-term about airborne infectious disease, rather than short-term, and a durable frame is a big part of that. 

It should definitely work, whether mounted inside or outside. It's probably worth a least a blog post or academic paper doing a head-to-head comparison. I will say that the Air Fanta 3Pro guy is an expert in water dynamics (@engineerwong is his handle), and applied his knowledge to air. The AirFanta 3Pro has some weird and surprising magic that makes it have such a high CFM. I don't fully understand how much is attributable to the filters, the higher RPM/static pressure fans, the interior space, all fans on the same side, or interior vs. exterior mounting. But he's racked up the best CFM/$, so he's one to learn from. I wanted to hit 300-400 CFM on a single unit, but I couldn't pull it off without making it more boxy.

**Comment by wanderinggnome648** [comment]
Also, when comparing the advertised results, keep in mind that all manufacturers report numbers which can vary. The best way I iterated was the same room, same setup, and just trying a lot of different designs quickly. It didn't matter whether my calculation for CADR was accurate, because I was just looking for relative comparisons. I'll likely send my box to the HouseFresh guy to put in his room for quick sanity check before I get too far down the road. It costs $$$ for the 3rd party test, but before I do that, I want to have high confidence I've optimized it well before doing so.

**Comment by raccettura** [comment]
Apparently part of that magic is the custom filters, higher pleat count = more surface area = less static pressure required.  Plus some apparently pretty loud fans.

**Comment by wanderinggnome648** [comment]
I haven't fully formed on opinion on making custom filters or not. They apparently aren't hard to have manufactured. But most vendors want to have custom filters and then mark them up a lot for on-going revenue. Schools end up tossing purifiers when the filter changes get expensive. So, I feel like for schools, the maintenance plan needs to be standardized/open.

**Comment by raccettura** [comment]
I’m of the opinion proprietary things suck, especially when standards exist.

I’d rather go taller. There’s no rule an air purifier needs to be square, and space above is unused.

**Comment by raccettura** [comment]
I was actually looking at 20x25, which seems pretty popular and 20x20 = 400 while 20x25 = 500.  That's a pretty substantial upgrade for slightly more height even with the same filter thickness.  Or 20x24x4.  The slightly taller height likely doesn't matter as you're not putting anything on top due to the fans being there.  It's in theory wasted space.

That's a lot more surface area to pull air through which should improve efficiency.  Also more surface area that has to be saturated before filters need to be replaced, which in theory lowers cost, and still standard sizes you can find with a variety of vendors https://filti.com/product/7500-home-filter/ . So no proprietary bullshit.
